docusaurus/docs/date-picker/input-date-picker.md

```diff
@@ -91,13 +91,21 @@ Limits which dates the user can navigate to and where events can go. Dates outsi
 `Type: boolean | undefined`  
 Flag indicating if the date format should be inside the components label.
 
+**withModal**
+`Type: boolean | undefined`  
+Flag indicating if the `datepicker` component is available from the input.
+
 **hasError**  
 `Type: boolean | undefined`  
-Flag indicating if the the component should display error styles.
+Flag indicating if the  component should display error styles.
 
 **hideValidationErrors**  
 `Type: boolean | undefined`  
-Flag indicating if the the component should hide error styles along with the `helperText` component displaying the error message.
+Flag indicating if the  component should hide error styles along with the `helperText` component displaying the error message.
```

Comment:
I looks like there are 2 spaces here, but good catch on the typo (the original typo was totally my fault)